crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -221,6 +221,63 @@ MAP
 ....xxxxxxx...@
 ENDMAP
 
+NAME:  nicolae_shoals_entry_flooding
+TAGS:  shoals_entry patrolling transparent
+KMONS: ON = water nymph
+KFEAT: O = enter_shoals
+SUBST: x = x`, ` = W., N = N .:80
+MAP
+ .........
+...xx`xx...
+..xxx`xxx..
+.xxxWWWxxx.
+.xxWNNNWxx.
+.``WNONW``.
+.xxWNNNWxx.
+.xxxWWWxxx.
+..xxx`xxx..
+...xx`xx...
+ .........
+ENDMAP
+
+NAME:  nicolae_shoals_entry_how_ready_are_you
+TAGS:  shoals_entry transparent no_pool_fixup
+KMONS: K = kraken
+KFEAT: K = w
+KFEAT: O = enter_shoals
+SUBST: M = wwx, _ = x_
```

Comment:
I think technically there's a very small chance that the whole box is enclosed by rock? Not sure if this matters. Players can usually boringly kite the kraken with a ranged attack from out of its range but that's a problem with almost every kraken vault so whatever.

Reply:
> I think technically there's a very small chance that the whole box is enclosed by rock?
Technically, although I guess in that case you could just call it a freebie. I might drop this one, though. I'm not super excited by it.